core/container/src/main/java/org/wildfly/swarm/internal/MavenArgsParser.java

```diff
@@ -0,0 +1,54 @@
+package org.wildfly.swarm.internal;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.StringTokenizer;
+
+/**
+ * Created by hbraun on 22.08.17.
+ */
+public class MavenArgsParser {
```

Comment:
Yea at the very least, I'd add a big TODO comment. The current usecase for this is our work on boosters, but anyone could use `-f` with their own Swarm project, totally unrelated to boosters.